app/assets/stylesheets/modules/home.scss

```diff
@@ -43,3 +43,12 @@
 #all_users.well {
   background-color: $white;
 }
+
+.nav > li > a.exhibit-inline {
```

Comment:
I'm not sure how to write this CSS better. The goal is for the Stanford "S" to be displayed on the same line as the link itself.